crates/bws/src/main.rs

```diff
@@ -516,12 +548,38 @@ async fn process_commands() -> Result<()> {
         | Commands::Delete {
             cmd: DeleteCommand::Secret { secret_ids },
         } => {
-            client
+            let result = client
                 .secrets()
                 .delete(SecretsDeleteRequest { ids: secret_ids })
                 .await?;
 
-            println!("Secret deleted correctly");
+            let mut secrets_success = 0;
+            let mut secrets_failed: Vec<(Uuid, String)> = Vec::new();
+
+            for secret in result.data.iter() {
+                match &secret.error {
+                    Some(error_message) => {
+                        secrets_failed.push((secret.id, error_message.to_owned()))
+                    }
+                    None => secrets_success += 1,
+                }
+            }
+
+            if secrets_success == 0 || secrets_success > 1 {
```

Comment:
```suggestion
            if secrets_success >= 0 {
```

Reply:
In this conditional I am purposefully excluding 1 for the grammar of the output. If `secrets_success == 1`, we print this message: `"{} secret deleted successfully."` (instead of `secrets` with the `s`)

Reply:
Is there a reason to print "0 secrets deleted successfully?"

Reply:
I wanted to make it clear to the user than none of the secrets they tried to delete were deleted. (Instead of just printing X secrets were deleted.) No other reason.